pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -176,7 +176,36 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 	// is safe.
 	if prev != nil {
 		// Check if the operator is put in the 'Network Migration' mode.
-		if _, ok := operConfig.GetAnnotations()[names.NetworkMigrationAnnotation]; !ok {
+		val, ok := operConfig.GetAnnotations()[names.NetworkMigrationAnnotation]
+
+		if ok && val != "start" && val != "prepare" {
+			log.Printf("invalid annotation value for migration: %s", val)
+		}
+
+		if !ok || val != "start" {
```

Comment:
if `!ok`, why do we want to continue checking for `"prepare"`?

Reply:
is it better to just say `if ok && val == "prepare" {`

Reply:
fixed.